### ISSUE TITLE
Feature request : train yolov4 with this repository for comparison

### ISSUE THREAD

**Issue by pfeatherstone** [ultralytics/yolov5#849]
@glenn-jocher Could you train yolov4 using this repository, i.e. same training loop, initial conditions, etc, for comparison against yolov3-spp and yolov5 models ?

**Comment by glenn-jocher** [comment]
@pfeatherstone yes, sort. We just need a yolov4.yaml to define the architecture. The augmentation, training, loss function etc from the paper would not be used though, we would use the yolov5 versions of all of those, but this would cool to compare. We did this for yolov3 already, but we need to update it since it was last trained with release v1.0.

**Comment by pfeatherstone** [comment]
It's ok using different augmentation and loss function to the paper. Using this repo, i.e. having uniform training loop, augmentation and loss function, allows us to compare model architecture only.
